events/syscall/utils_linux.go

```diff
@@ -91,3 +93,24 @@ func runAsUser(h events.Helper, username string, cmdName string, cmdArgs ...stri
 	}
 	return cmd.Run()
 }
+
+// Creates a temp directory and .ssh directory inside it.
+func CreateSshDirectoryUnderHome() (string, error) {
```

Comment:
```suggestion
func createSshDirectoryUnderHome() (string, error) {
```

Please, do not export internal utilities.

Reply:
ok Noted!